src/main/java/bookstore/entity/order/OrderItem.java

```diff
@@ -0,0 +1,40 @@
+package bookstore.entity.order;
+
+import bookstore.entity.book.Book;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "order_items")
+public class OrderItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "order_id")
+    private Order order;
+
+    @OneToOne
+    private Book book;
```

Comment:
The relation here is ManyToOne as Users can order the same book simultaniously. There may be multiple orders for one book. Also, it's better to specify FetchType.LAZY to avoid complex queries and n+1 problem

Reply:
And consider putting optional = false as orderitem for book can't exist without the book in db